app/models/customer.py

```diff
@@ -0,0 +1,22 @@
+from flask import current_app
+from app import db
+from datetime import datetime
+
+class Customer(db.Model):
+    customer_id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    registered_at = db.Column(db.DateTime)
+    # videos_checked_out_count = db.Column(db.Integer, default=0)
+    # rentals = db.relationship('Rental', back_populates='customer', lazy=True)
```

Comment:
You were on the right track of getting the videos check out as well as setting up the customer and rentals relationship.